packages/at_secondary_proxy/lib/src/secondary_proxy_server.dart

```diff
@@ -7,19 +7,21 @@ import 'secondary_connection_bridge.dart';
 class SecondaryProxyServer {
   static final String _certificateChainLocation = 'certs/fullchain.pem';
   static final String _privateKeyLocation = 'certs/privkey.pem';
-  static final String _trustedCertificateLocation = '/etc/cacert/cacert.pem';
+  static final String _trustedCertificateLocation = 'certs/cacert.pem';
```

Comment:
Where are we documenting the expectations here?

Reply:
Will make note in README